homeassistant/components/sensor/buienradar.py

```diff
@@ -110,9 +111,11 @@ def __init__(self, sensor_type, client_name):
         from buienradar.buienradar import (PRECIPITATION_FORECAST)
 
         self.client_name = client_name
-        self._name = SENSOR_TYPES[sensor_type][0]
         self.type = sensor_type
+        e_id = '{}_{}'.format(self.client_name, self.type)
+        self.entity_id = ENTITY_ID_FORMAT.format(e_id)
```

Comment:
Hi

I looked at this helper function; this probably is the helper function creating all entity_ids?
Using this function, I will end up with unique entity_ids but they will be the same as before, so there is no need to go this route....

I would rather use the same way of deriving entity_ids, than create an exception;
I will close this PR and create a new PR just for the remaining changes